packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -77,7 +77,7 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
 
     return (
       <div ref={this._rootElement} className={classNames.root} onFocusCapture={this._onFocusCapture}>
-        <div className={classNames.iconContainer} onClick={this._onClickFocus}>
+        <div className={classNames.iconContainer} onClick={this._onClickFocus} aria-hidden={true}>
```

Comment:
I'm concerned about this approach because this is an interactive element with an _onClickFocus handler, which will essentially make it unusable with assistive technology

Reply:
Hey guys thanks for the quick response - I made a gif of the before and the after behavior. 
(this is using edge and narrator)

![before](https://user-images.githubusercontent.com/2054644/41488963-13567b36-70a3-11e8-81e7-eaf1990b0d6a.gif)
^^ in the before gif I start with focus on the new entity button and use the down arrow keys to navigate through the menu bar - when it gets to the search box element it doesn't find the input and skips over it (You can still use tab to navigate to the search box but not using the arrow keys).

![after](https://user-images.githubusercontent.com/2054644/41489163-cc83048a-70a3-11e8-96eb-a7a368b572f4.gif)
^^ in the after gif I start with focus on the new entity button and use the down arrow keys to navigate through the menu bar and i'm able to enter the search input.

Reply:
Just duplicating offline notes:
Here is a CodePen doing something similar: https://codepen.io/anon/pen/rKGrKr?editors=0010

The CommandBar is working according to [ARIA `menubar` navigation guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_ex) in that arrow navigation does not leave the component. The SearchBar can be tabbed to, which makes sense given that it the next control in tab order.

I think this shows that in order to get the behavior you want, CommandBar and SearchBar have to be combined to some higher level order component, such as a `grid` role, that allows arrow navigation between all of the elements within that component.

Reply:
Can you render searchBar using `farItems` prop of CommandBar to include it within the CommandBar and enable arrow navigation?

Reply:
Arrow keys for keyboard navigation works fine here, the problem is with screen reader interaction. We currently render searchBar with the `farItems` prop as you describe. This is also why the elements in the search box aren't gaining visible focus, we are fixing a screen reader navigation problem.

Reply:
The CodePen wasn't set up that way...

Reply:
Ah sorry, the codepen must have had our old command bar

Reply:
@JasonGore Hey Jason, my bad a new version of the command bar was pushed on Friday. I just tested it however and the behavior is the same.

Reply:
CodePen with Search embedded highlighting issue for future reference: https://codepen.io/jasongore/pen/bKaWZJ

Reply:
Sorry for the churn here, but I'm curious, if you remove `aria-hidden` and add `role='presentation'`, does that also fix the issue?

Reply:
@JasonGore removing aria-hidden and adding role='presentation' has the same behavior as before

Reply:
Hmm, the subtle difference between them is that aria-hidden affects the entire hierarchy. I think either is ok in this case. I'll remove the block. Thanks!